packages/utils/telemetry-utils/src/errorLogging.ts

```diff
@@ -254,9 +257,13 @@ function overwriteStack(error: IFluidErrorBase | LoggingError, stack: string) {
  * False for any error we created and raised within the FF codebase, or wrapped in a well-known error type
  */
 export function isExternalError(e: any): boolean {
-    return !isValidLegacyError(e) ||
-        (e.getTelemetryProperties().untrustedOrigin === 1 &&
-         e.errorType === NormalizedExternalError.normalizedErrorType);
+    if (LoggingError.typeCheck(e)) {
+        if (NormalizedExternalError.typeCheck(e)) {
+            return e.getTelemetryProperties().untrustedOrigin === 1;
+        }
+        return false;
+    }
+    return !isValidLegacyError(e);
```

Comment:
What is legacy error type? I'd think it was used for staging and can be remove, right?

Reply:
I'll take a look when I'm back, please open a task in ADO and mention me in it so I can pull some data